web/beacon-app/src/features/onboarding/components/steps/workspace/form.tsx

```diff
@@ -0,0 +1,116 @@
+import { t } from '@lingui/macro';
+import { ErrorMessage, Form, FormikHelpers, FormikProvider } from 'formik';
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+import { stringify_org } from '@/utils/slugifyDomain';
+
+import { useNewWorkspaceForm } from '../../../useNewWorkspaceForm';
+import StepButtons from '../../StepButtons';
+
+const DOMAIN_BASE = 'https://rotational.io/';
+type WorkspaceFormProps = {
+  onSubmit: (values: any, helpers: FormikHelpers<any>) => void;
+  isDisabled?: boolean;
+  isSubmitting?: boolean;
+};
+const WorkspaceForm = ({ onSubmit, isSubmitting }: WorkspaceFormProps) => {
+  const formik = useNewWorkspaceForm(onSubmit);
+  const { getFieldProps, touched, setFieldValue, values } = formik;
+
+  useEffect(() => {
+    if (touched.workspace && values.workspace) {
+      console.log('touched.workspace', touched.workspace);
+      setFieldValue('workspace', stringify_org(values.workspace));
+    }
+    return () => {
+      touched.workspace = false;
+    };
+  }, [touched.workspace, setFieldValue, values, touched]);
+
+  return (
+    <FormikProvider value={formik}>
+      <Form className="mt-5 space-y-3">
+        <Fieldset>
+          <Span className="mt-[3px] font-medium">{DOMAIN_BASE}</Span>
+
+          <StyledTextField placeholder={t`your-workspace-url`} {...getFieldProps('workspace')} />
+
+          <div>
```

Comment:
Could the error message appear below the input instead of below where the slug will appear? This would make the error message's appearance more consistent with how error messages in the other steps will be displayed.

![Screenshot 2023-08-31 at 4 25 32 PM (2)](https://github.com/rotationalio/ensign/assets/94616884/4ac52094-2527-4c9a-9ac5-96e023d7b5b9)